rules-reviewed/camel3/camel2/xml-renamed-components.windup.xml

```diff
@@ -79,5 +79,185 @@
                 </hint>
             </perform>
         </rule>
+        <rule id="xml-renamed-components-00004">
+            <when>
+                <project>
+                    <artifact groupId="org.apache.camel" artifactId="camel-hdfs2" />
+                </project>
+            </when>
+            <perform>
+`                <hint title="`org.apache.camel:camel-hdfs2` artifact has been renamed" effort="1" category-id="mandatory" >
```

Comment:
there's a typo that prevents the XML file to be loaded and the test to be run.
```suggestion
                <hint title="`org.apache.camel:camel-hdfs2` artifact has been renamed" effort="1" category-id="mandatory" >
```

Reply:
oops my bad, removed the typo and everything runs smoothly, thanks !